.travis.yml

```diff
@@ -68,11 +71,11 @@ jobs:
         - export CXX=g++-4.8 CC=gcc-4.8
       before_cache:
         - ls -t regression/install | tail -n+2 | sed s@^@regression/install/@ | xargs rm -rf
-    - stage: prepare cache-verilator
+    - stage: set up verilator
       script:
-        - travis_wait 120 make verilator -C regression SUITE=none
-      before_install:
-        - export CXX=g++-4.8 CC=gcc-4.8
+        # This installs into /usr/local/bin/verilator
+        - curl -L -o verilator.deb https://github.com/sifive/verilator/releases/download/4.028-0sifive1/verilator_4.028-0sifive1_amd64.deb
+        - apt install -y verilator.deb
```

Comment:
can you do this with the travis package thing above instead of doing it directly here?

Reply:
by "package thing" i am referring to this list: https://github.com/chipsalliance/rocket-chip/blob/8af2fad8e5dcfc66d75c175d7999728d043adb20/.travis.yml#L22

Reply:
No, since this is an unofficial package. You can if you create your own apt repository, but I think that's kind of a lot of work.

I guess I could sign up for a [Launchpad](https://launchpad.net/ubuntu/+ppas) account, but that might be more trouble than necessary, since I'd have to learn how the Ubuntu package building actually works.